apps/evm/src/containers/ReadableActionSignature/getContractName.ts

```diff
@@ -48,11 +44,6 @@ const getContractName = ({ target, vTokens, tokens, chainId }: GetContractNameIn
         // Handle unique contracts
         return areAddressesEqual(contractAddress, target);
       }
-
-      // Handle SwapRouter contract
-      return Object.values(contractAddress).some(swapRouterAddress =>
-        areAddressesEqual(swapRouterAddress, target),
-      );
```

Comment:
AFAIU this wasn't even possible before. On line 35 we only consider the entries from `uniques` and `SwapRouter` is not there.